distribution/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/tests/patching/CARBON15878HazelcastAxis2ConfigTestCase.java

```diff
@@ -132,9 +132,9 @@ private CarbonServerCtrl(String carbonZipLocation)
             this.carbonHome = serverManager.setUpCarbonHome(carbonZipLocation);
 
             System.setProperty("javax.net.ssl.trustStore",
-                    Paths.get(this.carbonHome, "repository", "resources", "security", "wso2carbon.jks").toString());
+                    Paths.get(this.carbonHome, "repository", "resources", "security", "wso2carbon.p12").toString());
             System.setProperty("javax.net.ssl.trustStorePassword", "wso2carbon");
-            System.setProperty("javax.net.ssl.trustStoreType", "JKS");
+            System.setProperty("javax.net.ssl.trustStoreType", "PKCS12");
```

Comment:
Better if we can use the util method to set this

Reply:
This is only in the integration test code base.